### PR TITLE
Feature/authenticated user information

### PR DESCRIPTION
**Pull request by luismiguel0529**
## Summary
- Add method to view authenticated user
- Add Interface Authentication
## Trello Card
https://trello.com/c/6pWO41yc/27-acceder-a-la-informaci%C3%B3n-del-user-logueado

### CODE REVIEW

**Review by joredudiaz — DISMISSED** [review]

File: src/main/java/wolox/training/controllers/UsersController.java [hunk 1]
```diff
@@ -180,6 +192,21 @@ public void removeBook(@PathVariable Long id, @PathVariable Long bookid) {
...
+     * Method to view a autenticated user
+     *
+     * @return retunr autenticated user
+     */
+    @ApiOperation(value = "Method to view a autenticated user")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Authenticated user")
+    })
+    @GetMapping("/auth-username")
+    public ResponseEntity<String> authUsername() {
```
Comment (facucompe) [inline src/main/java/wolox/training/controllers/UsersController.java]: Return a JSON. Not a Plain string

Comment (facucompe) [reply src/main/java/wolox/training/controllers/UsersController.java]: Create an object, or return a User

**Review by facucompe — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/controllers/UsersController.java (hunk 1 above)
Comment (luismiguel0529) [reply src/main/java/wolox/training/controllers/UsersController.java]: corrected please validate

**Comment by luismiguel0529** [comment]
corrected please validate

**Review by facucompe — APPROVED** [review]

**Review by joredudiaz — APPROVED** [review]

**Pull request merged**